MANIFEST.in

```
@@ -4,3 +4,5 @@ include requirements.txt
 
 recursive-exclude * __pycache__
 recursive-exclude * *.py[co]
+
+graft iterseg/data
```

try changing this to:

```
graft src/iterseg/data
```

?

---

ah, that includes all the .zarr files which I don't think we want to ship. Maybe:

```suggestion
graft src/iterseg/data/*.pt
```

---

... still bringing in the zarrs. wth.

---

maybe move the zarrs (and csv) to a different dir, say, `_local_data/`?

---

ok, sorry for the wild goose chase — this seems to work:

```suggestion
recursive-include data *.pt
```

Still building but I see this line in the logs:

```
creating iterseg-0.1.0/data
```

which is encouraging!

---

Weird... The *source* distribution has it. But not the wheel. 🤔